wandb/integration/keras/keras.py

```diff
@@ -66,12 +78,10 @@ def is_generator_like(data):
 
 
 def patch_tf_keras():
-
+    from pkg_resources import parse_version
     from tensorflow.python.eager import context
 
-    from tensorflow import __version__ as tf_version
```

Comment:
i don't remember if there was a reason why we did it this way... but looks like it should be fine